api.bs

```diff
@@ -1959,12 +1987,18 @@ to ensure that [=privacy budgets=] are not inadvertently exceeded.
 
 ### Clearing Site Data ### {#clear-site-data}
 
-When clearing site data,
+When clearing site data at the request of a [=user=],
 but retaining browsing history,
 a [=user agent=] can [=map/set=] the value
 in the [=privacy budget store=] to 0 for all [=epochs=]
 for all affected [=sites=].
 
+When clearing site data at the request of a [=site=],
+through the use of the [:Clear-Site-Data:] header,
```

Comment:
Do we need to address the points from https://github.com/w3c/ppa/issues/125#issuecomment-2772458616?

Reply:
I think that the discussion there suggests - to me at least - that we need to keep that issue open and do a little more work.  This is probably enough for now, given that.